f2/src/forms/EvidenceInfoForm.js

```diff
@@ -99,8 +99,9 @@ export const EvidenceInfoForm = props => {
                   borderBottomWidth="2px"
                   borderBottomColor="grey.400"
                 >
-                  <Trans id="evidencePage.fileSize" />
-                  <P fontSize="sm">
+                  <P fontSize="md">
+                    <Trans id="evidencePage.fileSize" />
+                    <br />
                     <Text fontSize="sm" as="span" fontWeight="bold">
```

Comment:
Can you change this `fontSize="sm"` to `fontSize="md"` as well?

Reply:
fixed